cmd/init.go

```diff
@@ -47,11 +48,23 @@ See https://help.github.com/articles/searching-code/ for more details about the
 
 - Search target repos based on a Github Repo Search query.
 
-For example:
+To init all repos belonging to a specific org use --all-repos flag. Caution: you should run this command first to update all your local github repos.
+
+- cd into folder with all repos belonging to an org
+
+- for all repositories pull remote changes from github
+
+$ find . -type d -depth 1 -exec git --git-dir={}/.git --work-tree=$PWD/{} pull origin master \;
 
-$ mp init "org:Clever" --repo-search
+After that you can run the following command.
```

Comment:
why is it necessary to pull your local git repos? AFAICT --all-repos using the github API, and then from there further `microplane` operations do a fresh clone.

Reply:
I think `clone` was cloning the local version of github repos. As a result microplane planned changes for a number of repos that should't have been included.

Reply:
I don't see how that's possible... the call to `git clone` that microplane makes is always a git URL https://github.com/Clever/microplane/blob/937d048f6b63bf73927f9ffc713d6bb196315361/clone/clone.go#L35

Reply:
Hm weird. I will remove this note from the documentation.

I _think_ I ran into issues before without pulling changes from github. I worked on this a while ago so can't be sure anymore.

Reply:
If you don't remove your `mp/` dir between runs and re-run `mp` in the same directory, it'll see the old `mp/repo-name/clone` and whatnot. (I sometimes use this intentionally so I don't have to reclone, but it's often not what you want). Is that what happened?